corehq/tabs/tabclasses.py

```diff
@@ -1330,8 +1335,13 @@ def sidebar_items(self):
                     'title': _(TwoFactorProfileView.page_title),
                     'url': reverse(TwoFactorProfileView.urlname),
                 }
-            ))
+            ]]
         ]
+        if self.couch_user and self.couch_user.is_dimagi:
```

Comment:
my guess is less. not all dimagi email address are superusers, but i'd doubt many superusers are not dimagi email addresses